django_ses/__init__.py

```diff
@@ -193,28 +185,29 @@ def send_messages(self, email_messages):
 
             try:
                 response = self.connection.send_raw_email(
-                    source=source or message.from_email,
-                    destinations=message.recipients(),
-                    raw_message=dkim_sign(message.message().as_string(),
-                                          dkim_key=self.dkim_key,
-                                          dkim_domain=self.dkim_domain,
-                                          dkim_selector=self.dkim_selector,
-                                          dkim_headers=self.dkim_headers)
+                    Source=source or message.from_email,
+                    Destinations=message.recipients(),
+                    # todo attachments?
```

Comment:
@GitRon were you planning on doing this, is it worth highlighting in a changelog?

Reply:
Good point! I actually wanted to test attachements to make sure they would work but totally forgot about it! I'll check it asap.